test/releases/0.85.2.go

```diff
@@ -79,7 +79,7 @@ func init() {
 				ParentName: "secondary-dns",
 				ParentKind: "Deployment",
 				Name:       "secondary-dns",
-				Image:      components.CoreDNSImageDefault,
```

Comment:
build double check each patch release

it version should be according KSD tag

>= v0.0.11 ([PR](https://github.com/kubevirt/cluster-network-addons-operator/pull/1553/files))
[registry.k8s.io/coredns/coredns@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e](http://registry.k8s.io/coredns/coredns@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e)
else
[registry.k8s.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e](http://registry.k8s.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e)
or this which is the same if already there on components.go (strive to minimum changes in the current scope please)
[k8s.gcr.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e](http://k8s.gcr.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e) (edited)

Reply:
ACK. 
from my check:
v0.83.0 ksd version: v0.0.7
v0.83.1 ksd version: v0.0.8
v0.84.0 ksd version: v0.0.8
v0.85.0 ksd version: v0.0.9
v0.85.1 ksd version: v0.0.9
v0.85.2 ksd version: v0.0.9
v0.85.3 ksd version: v0.0.9
HEAD of release-0.85 branch: ksd version: v0.0.13

ACK. will apply accordingly.

Reply:
DONE